ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-interfaces.j2

```diff
@@ -25,4 +25,7 @@ interface {{ management_interface.name }}
 {%     if management_interface.ipv6_address is arista.avd.defined %}
    ipv6 address {{ management_interface.ipv6_address }}
 {%     endif %}
+{%     if management_interface.mac_address is arista.avd.defined %}
```

Comment:
@sugetha24 - when adding commands to to eos_cli_config_gen, we try and respect the cli order. The best way is to login to a DUT, and do: `show run all`

```eos
interface Management0
   no description
   no shutdown
   default load-interval
   mtu 1500
   logging event link-status use-global
   no description
   no shutdown
   default load-interval
   mtu 1500
   logging event link-status use-global
   no link-debounce
   no flowcontrol send
   no flowcontrol receive
   no speed
   no l2 mtu
   no l2 mru
   default logging event congestion-drops
   default unidirectional
   no traffic-loopback
   no mac-address
   default error-correction encoding
   no error-correction reed-solomon bypass
   phy media 25gbase-cr negotiation standard consortium ieee
   no mac timestamp
   snmp trap link-change
   vrf MGMT
```

The command should be generated between mtu and vrf, so move if block to line 16.

Reply:
Fixed